scio/src/main/scala/com/spotify/featran/scio/FeatranKryoRegistrar.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2018 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.featran.scio
+
+import com.esotericsoftware.kryo.{Kryo, Serializer}
+import com.esotericsoftware.kryo.io.{Input, Output}
+import com.spotify.scio.coders.KryoRegistrar
+import com.twitter.algebird.QTree
+import com.twitter.chill._
+
+@KryoRegistrar
+class FeatranKryoRegistrar extends IKryoRegistrar {
+  override def apply(k: Kryo): Unit = {
+    k.forClass(new QTreeSerializer)
+  }
+}
+
+private class QTreeSerializer extends Serializer[QTree[Any]] {
+  setImmutable(true)
```

Comment:
What is this for? From the doc it says `This causes {@link #copy(Kryo, Object)} to return the original object.`, is that called anywhere?

Reply:
Not in our code. But all `chill-algebird` serializers have this because most Algebird types are immutable by design.